scully/scully.ts

```diff
@@ -50,13 +52,16 @@ let _options = {};
   if (process.argv.includes('serve')) {
     port = options.path;
     console.log('starting static server...');
+    process.title = 'ScullyServer';
     checkChangeAngular(options.path);
     restartStaticServer();
   } else {
     const folder = join(scullyConfig.homeFolder, scullyConfig.distFolder);
 
     if (!existDistAngular(scullyConfig.homeFolder)) {
-      process.exit(0);
+      try {
```

Comment:
why is there a try here? proccess.exit doesn't usually fail?

Reply:
I will eliminate the try / catch ... the output works fine!
It is a test that I left unintentionally.